scripts/linters/js_ts_linter.py

```diff
@@ -1048,6 +1048,150 @@ def _check_comments(self):
+                    python_utils.PRINT(summary_message)
+                    python_utils.PRINT('')
+
+                dependency_injection_statement = (
+                    'private ' + camel_case_class_name + ': ' + class_name)
+                if not dependency_injection_statement in oppia_angular_root:
+                    total_error_count += 1
+                    summary_message = (
+                        'Please add the class %s to Oppia Angular'
+                        ' Root Component constructor in %s:'
+                        '  private %s: %s' % (
+                            class_name, oppia_angular_root_path,
+                            camel_case_class_name, class_name))
+                    summary_messages.append(summary_message)
+                    python_utils.PRINT(summary_message)
+                    python_utils.PRINT('')
+
+                static_variable_assignment = (
+                    'OppiaAngularRootComponent.' + camel_case_class_name +
+                    ' = ' + 'this.' + camel_case_class_name + ';'
+                )
+                if not static_variable_assignment in oppia_angular_root:
+                    total_error_count += 1
+                    summary_message = (
+                        'The static variable hasn\'t been assigned value'
+                        ' in Oppia Angular Root Component in %s:'
+                        '  OppiaAngularRootComponent.%s = this.%s' % (
+                            oppia_angular_root_path, camel_case_class_name,
+                            camel_case_class_name))
+                    summary_messages.append(summary_message)
+                    python_utils.PRINT(summary_message)
+                    python_utils.PRINT('')
+                class_name_in_quotes = '\'' + class_name + '\''
+                if not class_name_in_quotes in oppia_root_directive:
+                    total_error_count += 1
+                    summary_message = (
+                        'The class %s hasn\'t been added to ANGULAR_SERVICES'
+                        ' in  oppia-root.directive.ts. in %s' % (
+                            class_name, oppia_root_directive_path))
+                    summary_messages.append(summary_message)
+                    python_utils.PRINT(summary_message)
+                    python_utils.PRINT('')
+
+        with linter_utils.redirect_stdout(sys.stdout):
+            if self.verbose_mode_enabled:
+                python_utils.PRINT('----------------------------------------')
+            if total_error_count:
+                python_utils.PRINT('(%s files checked, %s errors found)' % (
+                    total_files_checked, total_error_count))
+                summary_message = (
+                    '%sOppiaAngularRootComponent linting failed, fix the'
+                    ' errors listed above' % (
+                        linter_utils.FAILED_MESSAGE_PREFIX))
+                summary_messages.append(summary_message)
```

Comment:
Move this and line 1187 to line 1190, I'm saying this to remove duplication.

Reply:
Done!